invenio_rdm_records/fixtures/data/vocabularies/resource_types.yaml

```diff
@@ -88,21 +88,57 @@
   tags:
     - depositable
     - linkable
-- id: publication-datamanagementplan
+- id: publication-conferenceproceeding
+  icon: file alternate
+  props:
+    csl: paper-conference
+    datacite_general: Text
+    datacite_type: Conference proceeding
+    openaire_resourceType: '4'
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/conferenceProceedings
+    schema.org: https://schema.org/ScholarlyArticle
+    subtype: publication-conferencepaper
+    type: publication
+  title:
+    en: Conference proceeding
+    de: Tagungsablauf
+  tags:
+    - depositable
+    - linkable
+- id: publication-outputmanagementplan
   icon: file alternate
   props:
     csl: report
     datacite_general: Text
-    datacite_type: Data management plan
-    openaire_resourceType: '9'
+    datacite_type: Output management plan
+    openaire_resourceType: '45'
     openaire_type: publication
     eurepo: info:eu-repo/semantics/technicalDocumentation
     schema.org: https://schema.org/CreativeWork
-    subtype: publication-datamanagementplan
+    subtype: publication-outputmanagementplan
     type: publication
   title:
-    en: Data management plan
-    de: Datenmanagementplan
+    en: Output management plan
+    de: Output-Management-Plan
```

Comment:
`de: Outputmanagementplan`

looks horrible but stays in line with `Datenmanagementplan`